EIPS/eip-IMPERSONATECALL.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: <undefined number>
+title: IMPERSONATECALL Opcode
+author: Sergio Demian Lerner (sergio.d.lerner@gmail.com)
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-09-24
+---
+
+### Overview
```

Comment:
```suggestion
## Abstract
```

`Overview` isn't a valid core EIP header. Please refer to EIP-1 for correct format. Also, note that section headers should be at a `##` depth, not `###`.